### PR TITLE
chore: Add ksaurabh4 profile

### PR DESCRIPTION
**Pull request by ksaurabh4**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1486"></a>

### CODE REVIEW

File: public/data/ksaurabh4.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      "description": "Started freelancing",
+      "url": " https://www.linkedin.com/in/ksaurabh4/"
+    },
+    {
+      "title": "Started Opensource",
+      "date": "June 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Created Video screen in an open source project animeworldz",
+      "url": "Anime Streaming App"
```
Comment (kumarsonsoff3) [inline public/data/ksaurabh4.json:53]: _Please **place a correct URL here** else **delete this line!**_ 


**Summary:** This code needs a Fix! 🛠

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
⚠ Please place a valid URL there in line no. 53.

And, unfortunately you've accidentally committed to the `package-lock.json` file, so also, **Revert** the changes from the `package-lock.json` file! 🛠

File: package-lock.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.68.8",
+      "version": "0.68.10",
```
Comment (emmalearnscode) [inline package-lock.json:9]:
```suggestion
      "version": "0.68.8",
```

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Please revert the changes to the package-lock.json file. Only your profile should be part of the pull request. If you need help please let us know.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

File: public/data/ksaurabh4.json (hunk 1 above)
Comment (ksaurabh4) [reply public/data/ksaurabh4.json]: Fixed

File: package-lock.json (hunk 2 above)
Comment (ksaurabh4) [reply package-lock.json]: Fixed

File: public/data/ksaurabh4.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "June 2020",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
+      "url": " https://www.linkedin.com/in/ksaurabh4/"
```
Comment (kumarsonsoff3) [inline public/data/ksaurabh4.json:45]: Small Fix! 

```suggestion
      "url": "https://www.linkedin.com/in/ksaurabh4/"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: public/data/ksaurabh4.json (hunk 3 above)
Comment (ksaurabh4) [reply public/data/ksaurabh4.json]: done

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kumarsonsoff3 — APPROVED** [review]

**Pull request merged**